v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/ICollectorPlugin.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SnapPluginLib
+{
+    public interface ICollectorPlugin
```

Comment:
streaming collector is not the same thing as standard collector. Why introducing single interface for them ?

Reply:
Streaming collector does not need to have Collect method.

Reply:
Good point. Extracted common IPuginInterface and removed Collect method from StreamingCollector